### DISCUSSION TITLE
Got a couple of questions, sorry for opening issue:

### DISCUSSION THREAD

**Discussion by InLoveWithMesh** [TomSchimansky/CustomTkinter#701] [General]
- How Can I align the Text in a Entry to the Left
- How is it possible to align a Text to the left and a image to the right of a button (at the same time
- If I want to replace the position where currently a Label is with a Progressbar? how can I set a object to an specific position before the current position (2 elements before the last one...)

**Comment by Akascape** [comment]
@InLoveWithMesh 
- Use `justify` to align the text in Entry. Eg: **justify="center"**
- For button, use `compound` to align the image to either left of right 
- In order to change position of any widget, you can change it's x-y position with place. Currently the progressbar does not have any command option so you have to update the label with a different method.
